lib/processor.js

```diff
@@ -286,7 +286,7 @@ function adjustBlock(block) {
 
         const out = {
             line: lineInCode + blockStart,
-            column: message.column + block.position.indent[lineInCode - 1] - 1
+            column: message.column + block.baseIndentText.length
```

Comment:
I don't think this change is valid in the case of "negative" indents, i.e. lines of code in a code block that are less indented than the opening fence. The parser previously gave us those on a per-line basis so we could make the correct adjustment. That shows up in the change to the "should translate indented column numbers" test, where one of the messages' columns (4) is now greater than the line length (2).